worker/dockerworker/dockerworker.go

```diff
@@ -92,6 +92,7 @@ func (d *dockerworker) StartWorker(run *runner.Run) (protocol.Transport, error)
 	// config from $DOCKER_WORKER_CONFIG.
 	mainJs := fmt.Sprintf("%s/src/bin/worker.js", d.wicfg.Path)
 	cmd := exec.Command("node", mainJs, "--host", "taskcluster-worker-runner", "production")
+	cmd.Env = os.Environ()
 	cmd.Env = append(cmd.Env, "DOCKER_WORKER_CONFIG="+d.wicfg.ConfigPath)
```

Comment:
If `DOCKER_WORKER_CONFIG` is already set in the environment, then this would cause it to be set twice. I'm not sure what would happen in this case. Perhaps better to remove any entries from the array that start with `DOCKER_WORKER_CONFIG=` before appending to it?

Reply:
The [docs](https://golang.org/pkg/os/exec/#Cmd) specify that the last appearance of an env var takes precedence:
>         // If Env contains duplicate environment keys, only the last
>         // value in the slice for each duplicate key is used.
so I think this is OK.